estate/models/estate_property.py

```diff
@@ -0,0 +1,19 @@
+from odoo import models,fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property Data Table"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date()
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float()
+    bedrooms = fields.Integer()
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(selection=[('north','North'),('south','South'),('east','east'),('west','West')])
```

Comment:
```suggestion
    garden_orientation = fields.Selection(selection=[('north', 'North'), ('south', 'South'), ('east', 'East'), ('west', 'West')])
```
read coding guidelines and follow them properly: https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html